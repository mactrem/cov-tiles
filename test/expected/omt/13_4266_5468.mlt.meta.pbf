F
	landcover
id" 
geometry* 
class" 	
subclass" 	°
waterway
id" 
geometry* 
class" 	
intermittent" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
brunnel" 	2
landuse
id" 
geometry* 
class" 	¾
place
id" 
geometry* 
class" 	
name" 	
name:ar" 	
name:br" 	
name:de" 	
name:en" 	
name:hu" 	
name:int" 	
name:ja" 	
name:la" 	

name:latin" 	
name:mk" 	
name:ru" 	
name:sr" 	
name:uk" 	
name:zh" 	
rank" ¤
poi
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" 
subclass" 	F
water
id" 
geometry* 
class" 	
intermittent" Æ
transportation_name
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
network" 	
ref" 	

ref:length" $
building
id" 
geometry* Ù
transportation
id" 
geometry* 
brunnel" 	
class" 	
layer" 
oneway" 
ramp" 
service" 	
surface" 	
subclass" 	
horse" 	
bicycle" 	
foot" 	