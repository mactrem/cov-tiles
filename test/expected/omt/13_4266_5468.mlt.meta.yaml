version: 1
featureTables {
  name: "landcover"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "waterway"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "landuse"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "place"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ar"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:br"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:hu"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ja"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:la"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:mk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:ru"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:sr"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:uk"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:zh"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "poi"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "water"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "transportation_name"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "network"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref:length"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "building"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
}
featureTables {
  name: "transportation"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "layer"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "oneway"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ramp"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "service"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "surface"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "horse"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "bicycle"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "foot"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
